### ISSUE TITLE
[Suggestion] Ability to rate programs

### ISSUE THREAD

**Issue by nshusa** [solus-project/solus-sc#193]
You can rate programs 1-5 stars. This will let other people know how well the community likes these programs. Programs with low rating can indicate its outdated or simply doesn't work. Whereas programs that are rated 5 stars can indicate it works really well with Solus.

Here's an example from another Distro 
![https://i.imgur.com/vGuPR3w.png](https://i.imgur.com/vGuPR3w.png)

**Comment by Girtablulu** [comment]
Well out dated or buggy software on solus should be reported and not rated! I only can see it as a general information what the community things about the software

**Comment by nshusa** [comment]
@Girtablulu Yeah because the maintainer is gonna wanna go through all those reports and check if the software really works. Have fun with that.

**Comment by baimafeima** [comment]
I don't think user-based ratings (1-5) of software is a good idea. Instead I'd suggest to utilize a score card: #126 
Admittedly, such a score card project takes a lot of time but if maintained by the Solus community rather than the developers it could be feasible. Alternatively, an option to sort or **tag** software packages with various labels by their degree of usability and/or complexity seems to me a more computing-friendly approach: _basic_, _intermediate_, _advanced_ or _easy_, _medium_, _hard_. See also: #57
